netbox/templates/inc/user_menu.html

```diff
@@ -1,11 +1,18 @@
 {% load i18n %}
 {% load navigation %}
 
+{% if 'help-center' in settings.RELEASE.features %}
```

Comment:
I don't see features in release.yaml? Is that added in enterprise, cloud and can we get it documented somewhere (and what features there are) if it isn't already?

Reply:
This was part of #15908. We didn't document the `features` as it should only be used internally.